Sources/Plasma/Apps/plPythonPack/main.cpp

```diff
@@ -426,61 +342,69 @@ void PackDirectory(const plFileName& dir, const plFileName& rootPath, const plFi
     }
 
     s.Close();
+    ST::printf(out, "\nPython Package written to {}\n", pakName);
 
     PythonInterface::finiPython();
 }
 
 void PrintUsage()
 {
-    fputs("Usage:\n", stdout);
-    fputs("plPythonPack [directory to pack...]\n", stdout);
-    fputs("NOTE: the directory to pack must have full system and plasma dirs and\n", stdout);
-    fputs("      must be a relative path to the current working directory\n", stdout);
+    ST::printf(out, "The Python Pack Utility\n");
+    ST::printf(out, "Usage:\n");
+    ST::printf(out, "plPythonPack [options] <directory to pack...>\n");
+    ST::printf(out, "NOTE: the directory to pack must have full system and plasma dirs.\n");
+    ST::printf(out, "\nAvailable options:\n");
+    ST::printf(out, "\t-q\tQuiet  - Only print errors\n");
+    ST::printf(out, "\t-h\tHelp   - Print this help\n");
```

Comment:
Is there any reason `PrintUsage()` would want to be redirected?

Reply:
Other than copy & paste, I can't think of anything.  In fact it feels like it would be pretty nonsensical to be asked for help and then keep quiet about it.